src/Maps/Map.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <bits/stdc++.h>
+#include <string>
+#include <SFML/Graphics/RenderWindow.hpp>
+#include <SFML/Graphics/Texture.hpp>
+#include <SFML/Graphics/Sprite.hpp>
+#include <SFML/Graphics/View.hpp>
+#include <RealVec.h>
+#include <Texture/Texture.h>
+#include <StartPosition.h>
+#include <Drawable.h>
+#include <GameObjects/ObjectTypes.h>
+#include <Object.h>
+#include "CheckpointPosition.h"
+
+/**
+ * Map class.
+ * Contains "physical dimensions".
+ * It defines pixel/meters scale.
+ */
+class Map : public Drawable {
+public:
+    Map(std::string name, std::string description, RealVec size, TexturePtr view, TexturePtr friction,
+            TexturePtr minimap, std::vector<CheckpointPosition> &&checkpoints, std::vector<ObjectPtr> &&objects,
```

Comment:
Być może lepiej byłoby przekazywać checkpointy i objecty przez wartość skoro to tylko ptr